libcxx/utils/generate_feature_test_macro_components_v2.py

```diff
@@ -0,0 +1,825 @@
+#!/usr/bin/env python
```

Comment:
I think it would be nice to store the FTM data in a pure JSON file, and have that script use that data to generate the FTM, tests, etc.